core/src/snapshot_packager_service.rs

```diff
@@ -97,6 +98,14 @@ impl SnapshotPackagerService {
             }
         }
 
+        // Write version file
+        {
+            use std::io::Write;
+            let snapshot_version = env!("CARGO_PKG_VERSION");
+            let mut f = std::fs::File::create(staging_version_file)?;
+            f.write_all(&snapshot_version.to_string().into_bytes())?;
```

Comment:
I have strong peculiarity for wanting a newline for all of text files. :)

Reply:
I can abide that!